connection.go

```diff
@@ -29,7 +33,7 @@ type Options struct {
 }
 
 type Greeting struct {
-	Version []byte
+	Version uint32
```

Comment:
Since this property is supposed to hold the value of versionID function call, wouldn't it make more sense to make VersionID public too?

Reply:
Until tarantool version isn't public there is no sense to make versionID public.